code/api/src/config/database.json

```diff
@@ -1,6 +1,6 @@
 {
   "development": {
-    "username": "tashiadavis",
+    "username": "alice",
```

Comment:
Hopefully we can figure something out so we don't keep having to do this every PR! 👍